src/panels/lovelace/types.ts

```diff
@@ -6,3 +8,7 @@ export interface LovelaceCard {
   getCardSize(): number;
   setConfig(config: LovelaceConfig): void;
 }
+
+export interface LovelaceCardElement extends HTMLElement {
```

Comment:
`LovelaceCardElement` is a `LovelaceCard`. If you rebase on `master`, this change has already been applied https://github.com/home-assistant/home-assistant-polymer/blob/c05b77961ebbcaae385bcb53689f2992114cc0cd/src/panels/lovelace/types.ts#L7-L11